pkgs/by-name/tl/tldx/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "tldx";
+  version = "1.0.0";
```

```suggestion
  version = "1.0.0";
  commit = "9085db6bf292b243e22ee13442a8759acebbfcfd";
```
This can be used to populate `main.commit`.